pkg/monitor/serialization/serialize.go

```diff
@@ -144,9 +144,10 @@ func IntervalsToFile(filename string, intervals monitorapi.Intervals) error {
 func EventsIntervalsToJSON(events monitorapi.Intervals) ([]byte, error) {
 	outputEvents := []EventInterval{}
 	for _, curr := range events {
-		if curr.From == curr.To && !curr.To.IsZero() {
+		if !curr.Display || (curr.From == curr.To && !curr.To.IsZero()) {
 			continue
 		}
+
```

Comment:
This logic is still wrong. Unported events will be Display = false by default due to go zero values. I think I need to remove Display here entirely, we keep the curr.From and curr.To check.